src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,9 +1,29 @@
+import { useState } from "react";
 import { Movie } from "../../models/Movie.model";
+import DeleteMovie from "../DeleteMovie/DeleteMovie";
+import EditMovie from "../EditMovie/EditMovie";
 import classes from "./MovieCard.module.css";
 
 const MovieCard: React.FC<Movie> = ({ id, name, genre, year, poster }) => {
+  const [menuVisible, setMenuVisible] = useState(false);
+
   return (
     <div className={classes.card}>
+      <div className={classes.cardMenu}>
+        {!menuVisible && (
+          <button className={classes.closeMenuButton} onClick={() => setMenuVisible(true)}>
+            ...
+          </button>
+        )}
+        {menuVisible && (
+          <div className={classes.menuContainer}>
+            <button onClick={() => setMenuVisible(false)}>Close</button>
+            <EditMovie />
+            <DeleteMovie />
+          </div>
+        )}
+      </div>
```

Comment:
Consider this same code a bit different:
```suggestion
      <div className={classes.cardMenu}>
        {!menuVisible ? (
          <button className={classes.closeMenuButton} onClick={() => setMenuVisible(true)}>
            ...
          </button>
        ) : (
          <div className={classes.menuContainer}>
            <button onClick={() => setMenuVisible(false)}>Close</button>
            <EditMovie />
            <DeleteMovie />
          </div>
        )}
      </div>
```